src/Analyzer.JsonRuleEngine.UnitTests/JsonRuleEngineTests.cs

```diff
@@ -277,57 +275,44 @@ private void AssertEvaluationsAndResultsAreAsExpected(IEvaluation evaluation, in
                 }
             }]", DisplayName = "Invalid regex pattern")]
         [ExpectedException(typeof(JsonRuleEngineException))]
-        public void EvaluateRules_InvalidRule_ExceptionIsThrown(string invalidRule)
+        public void Create_InvalidRules_ExceptionIsThrown(string invalidRule)
         {
-            // Arrange
-            var template =
-                JObject.Parse(
-                @"{
-                    ""$schema"": ""https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#"",
-                    ""resources"": [
-                        {
-                            ""type"": ""Microsoft.ResourceProvider/resource0"",
-                            ""properties"": {
-                                ""somePath"": ""someValue"",
-                                ""someOtherPath"": ""someOtherValue""
-                            }
-                        }
-                    ]
-                }");
-
-            string expectedFileId = "MyTemplate";
-
-            TemplateContext templateContext = new TemplateContext
-            {
-                OriginalTemplate = template,
-                ExpandedTemplate = template,
-                IsMainTemplate = true,
-                TemplateIdentifier = expectedFileId
-            };
+            // Act
+            JsonRuleEngine.Create(invalidRule, t => null);
+        }
 
-            // Setup mock line number resolver
-            var mockLineResolver = new Mock<ILineNumberResolver>();
+        [ExpectedException(typeof(JsonRuleEngineException))]
```

Comment:
Good catch on the jsonLineNumberResolverBuilder.

I'm not testing ParseRuleDefinitions directly because it's private.  Because of that, the "unit" of work being tested starts at a higher level, in this case through the `Create` method.

Reply:
Oh, I did already have a test for that - line 306.

Reply:
Ah, see the new open comment. The functions are missing `[TestMethod]` so aren't actually part of the test suite.